src/app/shared/search/search-filters/search-filters.component.html

```diff
@@ -1,7 +1,8 @@
-<h3>{{"search.filters.head" | translate}}</h3>
+<label class="h3">{{filterLable+'.filters.head' | translate}}</label>
```

Comment:
This change is also throwing accessibility warnings in WAVE (again the warning is this is an "orphaned form label").  I don't understand why we need to change this from an `<h3>` to a `<label class="h3">`.  Could you either change this back to an `<h3>` or explain why this change was made?  We need to fix the accessibility here before we can merge.